src/components/letter/FolderSelect.tsx

```
@@ -1,59 +1,92 @@
-
   return (
-    <div className="fixed inset-0 z-50 flex justify-center">
-      <div className="relative w-full max-w-[440px] min-h-screen flex items-center justify-center">
-        <button
-          type="button"
-          onClick={onClose}
-          className="absolute inset-0 bg-black/40"
-        />
+    <BottomSheet open={open} onClose={onClose} className="max-h-[369px] px-[20px]">
+      <div className="mt-[20px] flex w-full flex-col overflow-y-auto">
+          {/* 선택 없음(전체) */}
```

ESLint

[error] Delete `··`

(prettier/prettier)